### ISSUE TITLE
Bug: core 2022.3.2 

### ISSUE THREAD

**Issue by bmk1964** [JurajNyiri/HomeAssistant-Tapo-Control#151]
Tapo camera control gives error and don't start after core 2932.3.2

**Comment by JurajNyiri** [comment]
This has been fixed for [11 days now](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/3.4.5). Please refrain from reporting issues if you are using outdated versions. 

There are also 3 duplicates already:

https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/150
https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/149
https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/148

**Comment by Leah96xxx** [comment]
You may have uploaded a fix, however 3.4.5 cannot be downloaded through HACS

**Comment by JurajNyiri** [comment]
You are correct, I did not notice that. It should be fixed now.

**Comment by Leah96xxx** [comment]
It works now. Hopefully that stops people nagging you about it 🙂

**Comment by JurajNyiri** [comment]
@Leah96xxx Yeah I was wondering how come so many people did not update and also ignored all the closed issues.
In any case it [motivated me](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/159) to learn Github Actions and set up a bot that closes any issues not using the template and adding a separate section in there to make sure people used search before asking.
![image](https://user-images.githubusercontent.com/5785999/157114810-560adede-622d-46fd-8f64-a97e5d5d9849.png)

So it was all for the better after all!
